[faq] Add tracks recording in background section troubleshooting (#259)
organicmaps/organicmaps.github.io: content/faq/bookmarks/android-track-recording/index.pt.md

---

> title: Porque é que as faixas não são gravadas de forma fiável em segundo plano no Android?

Reviewer: ```suggestion
title: Porque é que os trajetos não são gravados de forma fiável em segundo plano no Android?
```

> description: As configurações padrão de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou matar o aplicativo Organic Maps em segundo plano.

Reviewer: ```suggestion
description: As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou encerrar a aplicação Organic Maps em segundo plano.
```

updated: "2024-10-05"

taxonomies:
  faq: ["Favoritos e trilhas"]

extra:
  order: 30
---

> As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou eliminar a aplicação Mapas orgânicos em segundo plano.

Reviewer: ```suggestion
As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou encerrar a aplicação Organic Maps em segundo plano.
```

> Isso é especialmente verdadeiro para versões modernas do Android:

Reviewer: ```suggestion
Isso acontece especialmente nas versões mais modernas do Android:
```

- Android 14
- Android 13
- Android 12
- Android 11

> Os passos exactos sobre como fazer com que os Mapas orgânicos (e outras aplicações) funcionem em segundo plano estão listados aqui: https://dontkillmyapp.com/

Reviewer: ```suggestion
Os passos exatos sobre como fazer com que o Organic Maps (e outras aplicações) funcionem em segundo plano estão listados aqui: https://dontkillmyapp.com/
```
